src/main/java/org/graylog/integrations/aws/service/KinesisService.java

```diff
@@ -188,12 +189,14 @@ public StreamsResponse getKinesisStreamNames(String regionName, String accessKey
         // Create retryer to keep checking if more streams exist.
         final Retryer<Boolean> retryer = RetryerBuilder.<Boolean>newBuilder()
                 .retryIfResult(b -> Objects.equals(b, Boolean.TRUE))
+                .retryIfExceptionOfType(LimitExceededException.class)
```

Comment:
Possibly , we need to think about other ExceptionTypes, when we may want to retry.

Reply:
Thanks for taking a look and for the suggestion. Perhaps in the future we can expand the list to other exceptions if we know which ones they are.

Since this PR is specifically for rate limiting, I propose that we include only the `LimitExceededException` in this PR.